src/router/middleware/ifLoggedInDashboardElseNext.js

```diff
@@ -0,0 +1,7 @@
+export default function ifLoggedInDashboardElseNext({ next, store }) {
```

Comment:
The Name of this file  should be `authMiddleware.js`

Reply:
Asta nu mai folosește la nimic, e lăsată doar că un exemplu că se pot implementa